python-main.js

```diff
@@ -1164,9 +1164,13 @@ function web_editor(config) {
         });
         $("#command-help").click(function (e) {
             // Show help
+            if($(window).width() > 620){
```

Comment:
The indentation in the two lines before should be increased since they are now inside this `if` statement.